test/obb/server/controllers/mock_request.clj

```diff
@@ -0,0 +1,12 @@
+(ns obb.server.controllers.mock-request
+  (:use clojure.test)
+  (:require [obb.server.controllers.index :as index]
+            [obb.server.core :as core]
+            [clojure.edn :as edn]
+            [ring.mock.request :as mock]))
+
+(defn http-get
+  "Mocks a HTTP GET request. Returns the body as edn"
+  [path]
+  (let [response (core/app (mock/request :get "/"))]
+    (assoc response :body (edn/read-string (:body response)))))
```

Comment:
There are a couple of things about this component that I think could be improved:

* I think the function could be a more general and also receive the request type. I imagine this won't be a lot different when we have `:put` or `:post` requests.

* I think the name is a bit misleading, since the request is not actually mocked, right? I know this is the terminology used in the library (`ring.mock`) but this is not truly a mock is it? This is more like a helper that allows to easily create request maps, and in our case, allows us to parse the response for the tests.
I am not sure what would be the best name but I think it would be cool to have something that stated what we want, instead of what we are doing. Maybe `parsed-response`?

```clj
(parsed-response :get "/")
```